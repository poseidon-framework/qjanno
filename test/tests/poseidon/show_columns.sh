qjanno "SELECT * FROM data/Schmid_2028/Schmid_2028.janno" --showColumns
qjanno "SELECT * FROM d(data)" --showColumns