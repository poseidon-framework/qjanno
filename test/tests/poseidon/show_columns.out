.-------------.------------------------------------.--------------------------.
|   Column    |                Path                |    qjanno Table name     |
:=============:====================================:==========================:
| source_file | data/Schmid_2028/Schmid_2028.janno | dataSchmid2028Schmid2028 |
| Genetic_Sex | data/Schmid_2028/Schmid_2028.janno | dataSchmid2028Schmid2028 |
| Group_Name  | data/Schmid_2028/Schmid_2028.janno | dataSchmid2028Schmid2028 |
| Poseidon_ID | data/Schmid_2028/Schmid_2028.janno | dataSchmid2028Schmid2028 |
'-------------'------------------------------------'--------------------------'
.-------------.---------.-------------------.
|   Column    |  Path   | qjanno Table name |
:=============:=========:===================:
| source_file | d(data) | ddata             |
| AddCol1     | d(data) | ddata             |
| AddCol2     | d(data) | ddata             |
| AddCol3     | d(data) | ddata             |
| Genetic_Sex | d(data) | ddata             |
| Group_Name  | d(data) | ddata             |
| Poseidon_ID | d(data) | ddata             |
| Publication | d(data) | ddata             |
'-------------'---------'-------------------'
